spring-pulsar/src/main/java/org/springframework/pulsar/listener/DefaultPulsarMessageListenerContainer.java

```diff
@@ -172,11 +172,7 @@ private final class Listener implements SchedulingAwareRunnable {
 
 		private final boolean isBatchListener = this.containerProperties.isBatchListener();
 
-		private final boolean isManualAck = this.containerProperties.getAckMode().equals(AckMode.MANUAL);
-
-		private final boolean isBatchAck = this.containerProperties.getAckMode().equals(AckMode.BATCH);
-
-		private final boolean isRecordAck = this.containerProperties.getAckMode().equals(AckMode.RECORD);
+		private final AckMode ackMode = this.containerProperties.getAckMode();
 
 		private final boolean isSharedSubscription = this.containerProperties.getSubscriptionType()
```

Comment:
I did not call it out explicitly, but for consistency do you mind doing the same for subscritpion type?